make/kube

```diff
@@ -3,19 +3,27 @@
 set -o errexit -o nounset
 
 GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
-
-. "${GIT_ROOT}/.envrc"
-
 cd "${GIT_ROOT}"
+source .envrc
 
 ENV_FILES="$(echo bin/settings/*.env bin/settings/kube/*.env)"
 
-# TODO: We currently skip memory limits for development purposes
+CREATE_HELM_CHART=false
+BUILD_TARGET=kube
 
-fissile build kube \
-    --kube-output-dir="${GIT_ROOT}/kube" \
+if [ "${1:-}" = "helm" ]; then
+    CREATE_HELM_CHART=true
+    BUILD_TARGET=helm
+    ENV_FILES="$(echo bin/settings/certs.env bin/settings/kube/ca.env)"
+fi
+
+rm -rf ${BUILD_TARGET}
+
+# TODO: We currently skip memory limits for development purposes
+fissile build ${BUILD_TARGET} \
+    --output-dir=${BUILD_TARGET} \
     --defaults-file="$(echo "${ENV_FILES}" | tr ' ' ,)" \
-    --use-memory-limits=false \
+    --use-memory-limits=false
 
 # This is a small hack to make the output of make kube be compatible with K8s 1.6
-perl -p -i -e 's@extensions/v1beta1@batch/v1@' kube/bosh-task/*.yml
+perl -p -i -e 's@extensions/v1beta1@batch/v1@' $(grep -rl "kind: Job" ${BUILD_TARGET})
```

Comment:
Quotes around `${BUILD_TARGET}` (yes I know there are no spaces; it's a shellcheck thing)